crates/bitmex-stream/src/lib.rs

```diff
@@ -68,7 +67,7 @@ fn subscribe_impl<const N: usize>(
                 .duration_since(UNIX_EPOCH)?
                 .add(Duration::from_secs(5))
                 .as_secs();
-            let signature = credentials.sign(Method::GET, expires, &Url::parse(url.as_str())?, "");
+            let signature = credentials.sign(Method::GET, expires, &url, "");
```

Comment:
Does this still work? Before we passed in a `Url` and called `.path()` on it, returning only the `path` of the url, now you pass in the whole `url`.

Reply:
I tested it for you, it does not 😅
```
2022-11-15T23:36:36.300115Z  INFO authenticated_api: {"status":401,"error":"Signature not valid.","meta":{},"request":{"op":"authKeyExpires","args":["**********--oa9",1668555400,"XXXXXXX"]}}
```